### ISSUE TITLE
from_entries returns null for empty array

### ISSUE THREAD

**Issue by mantljosh** [01mf02/jaq#38]
When the array is empty, from_entries returns null instead of an empty object:

```
$ jaq "from_entries" <<(echo "[]")
null
```

This means it is not a true inverse of `to_entries` which can be problematic if you're not sure whether the object will have any keys, but you need to ensure the output remains an object

```
$ jaq "to_entries | from_entries" <<(echo "{}")
null
```

**Comment by pkoppstein** [comment]
For reference, jq and gojq get it right.

**Comment by 01mf02** [comment]
Thanks, this is corrected in d6276cd.
